Changelog.md

```diff
@@ -3,6 +3,14 @@
 __New Features__
 - **Breaking change**: Updates to newer proposed HPXML v4.0:
   - Replaces `PortableHeater` and `FixedHeater` with `SpaceHeater`.
+- Allows simulating whole multifamily (MF) buildings, where each dwelling unit is described by a separate `Building` element.
+  - Allows `NumberofUnits` to be used as a multiplier on dwelling unit simulation results to reduce simulation runtime.
+  - Adds a `building_id` argument to the HPXMLtoOpenStudio, BuildResidentialHPXML, and BuildResidentialScheduleFile measures for whole MF HPXML files.
```

Comment:
There is no `building_id` argument for `BuildResidentialHPXML`, right?

Reply:
Oops. Good point.